tests/fuzz/fuzz_rpc/initialisation.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "crypto/hash.h"
+#include "rpc/core_rpc_server.h"
+#include <fuzzer/FuzzedDataProvider.h>
+
+// Define templates for dummy protocol object
+template class nodetool::node_server<cryptonote::t_cryptonote_protocol_handler<cryptonote::core>>;
+
+struct DummyProtocol : public cryptonote::i_cryptonote_protocol {
```

Comment:
We are a little inconsistent, but most of the code uses snake case instead of camel case. I guess since this is going into fuzz, this matters less.